src/views/answerView.js

```diff
@@ -4,10 +4,38 @@
  * Create an Answer element
  * @returns {Element}
  */
-export const createAnswerElement = (key, answerText) => {
+
+let score = 0
```

Comment:
I think this should be in our `quizData` object instead of a global variable in this file

Reply:
done